### PR TITLE
fix: asset bundle conversion dependency issues

### PR DESCRIPTION
**Pull request by BrianAmadori**
In the asset bundle conversor context, GLBs with buffers as external dependencies (used mainly for animations) were injected in the `PersistentAssetCache` with a wrong ID, leading to conversion failure. 

I added some fixes so the tests pass, but another refactor PR will be created to make the tests internals more mockeable in order to expand the coverage.

### CODE REVIEW

File: unity-client/Assets/Batch/AssetBundleMenuItems.cs [hunk 1]
```diff
@@ -55,6 +57,15 @@ static void DumpAreaToMax(AssetBundleBuilder builder, int x, int y)
             builder.DumpArea(new Vector2Int(x, y), new Vector2Int(10, 10), (error) => DumpAreaToMax(builder, nextX, nextY));
         }
 
+        [MenuItem("Decentraland/Asset Bundle Builder/Dump Tomb Chaser")]
```
Comment (AjimenezDCL) [inline unity-client/Assets/Batch/AssetBundleMenuItems.cs:60]: I guess this is debug code to be able to quickly check that the fix worked with the affected scene. Is there any reason to keep it?

File: unity-client/Assets/Batch/AssetBundleBuilder.cs [hunk 2]
```diff
@@ -55,7 +56,7 @@ public enum ErrorCodes
 
         public Dictionary<string, string> hashLowercaseToHashProper = new Dictionary<string, string>();
 
-        internal ContentServerUtils.ApiEnvironment environment = ContentServerUtils.ApiEnvironment.ORG;
+        internal ContentServerUtils.ApiTLD tld = ContentServerUtils.ApiTLD.ORG;
```
Comment (AjimenezDCL) [inline unity-client/Assets/Batch/AssetBundleBuilder.cs:59]: What does TLD stand for? Not a big fan of acronyms unless they are plain obvious

**Review by AjimenezDCL — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Batch/AssetBundleBuilder.cs (hunk 2 above)
Comment (BrianAmadori) [reply unity-client/Assets/Batch/AssetBundleBuilder.cs:59]: TLD = Top level domain, the reason for the change is that same term is used for this purpose at kernel, so I thought it'd be better to improve terms ubiquity between both projects.

File: unity-client/Assets/Batch/AssetBundleMenuItems.cs (hunk 1 above)
Comment (BrianAmadori) [reply unity-client/Assets/Batch/AssetBundleMenuItems.cs:60]: If you look at this file, there are several similar menu items that dump several locations. I use them from time to time for testing and deleting it'd imply having to rewrite it each time I want to test this scene.

A better approach'd be to build a custom editor window for this purpose, but for now this is the only way I have to test important scenes.

**Pull request merged**